_docs/README.md

```diff
@@ -40,9 +40,19 @@ where you want the generated code to go:
 ```
 boilerplate --template-folder /home/ubuntu/website-boilerplate --output-folder /home/ubuntu/website-output
 
-Enter the value for 'Title': Boilerplate Example
-Enter the welcome text for the website: Welcome!
-Should the website show the logo? (y/n) (default: "y"): y
+Title
+
+  Enter a value [type: string]: Boilerplate Example
+
+WelcomeText
+  Enter the welcome text for the website
+
+  Enter a value [type: string]: Welcome!
+
+ShowLogo
+  Should the website show the logo?
+
+  Enter a [type: bool]: true
```

Comment:
I like this new UX. The `description` can be as long as you want, and the prompt is typed. Nice!